src/Visithor/Command/GoCommand.php

```diff
@@ -137,7 +138,16 @@ protected function executeVisithor(
         array $config,
         $format
     ) {
-        $client = new GuzzleClient();
+        // check client in config
+        if (isset($config['defaults']['client']) && class_exists($config['defaults']['client'])) {
+            $clientClass = $config['defaults']['client'];
+            $client = new $clientClass();
+        }
+        // default to Guzzle
+        else {
```

Comment:
Can you use [PSR-2](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md) please?